app/utilities/schema.py

```diff
@@ -45,18 +46,18 @@ def load_schema_file(schema_file, language_code=None):
         raise e
 
 
-def _get_schema_file_path(schema_file, language_code):
-    if language_code != 'en':
-        schema_dir = os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
-    else:
-        schema_dir = settings.EQ_SCHEMA_DIRECTORY
+def _get_schema_path(language_code='en'):
+    return os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
 
+
+def _get_schema_file_path(schema_file, language_code):
+    schema_dir = _get_schema_path(language_code)
     return os.path.join(schema_dir, schema_file)
 
 
-def available_json_schemas():
+def available_json_schemas(language_code='en'):
```

Comment:
We don't need to have a `language_code` parameter here as we always load the list of available schemas from the default language directory.